Recrutify/Recrutify.Host/CustomAttribute/ParseJsonToDict.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Collections.Generic;
+using Newtonsoft.Json;
+
+namespace Recrutify.Host.CustomAttribute
```

Comment:
Put to folder Infrastructure/Models, also rename your folder UserServices to Identity, and namespace too